dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/data/PutJobData.kt

```diff
@@ -117,6 +125,15 @@ data class PutJobData(
         val parent = item.second.parent
         val paths = item.second.toFile().walk(FileWalkDirection.TOP_DOWN)
                     .filter { !(it.isDirectory && Files.list(it.toPath()).use { f -> f.findAny().isPresent }) }
+                    .filter {
+                     try {
+                        it.toPath().toRealPath()
+                        true
+                    } catch (e: java.nio.file.NoSuchFileException) {
```

Comment:
Why did you specify the absolute class name here?  Shouldn't the import at the top eliminate the need for this?

Reply:
When I use just the class name it ends up with the kotlin.io.NoSuchFileException which doesn't match what's thrown.

Is there a better way?

Reply:
Explicitly specify the import at the top.  If you don't you'll get the kotlin one.